compiler/frontend/src/org/jetbrains/kotlin/psi/psiUtil/ktPsiUtil.kt

```diff
@@ -454,3 +454,17 @@ fun checkReservedPrefixWord(sink: DiagnosticSink, element: PsiElement, word: Str
     }
 }
 
+fun KtNameReferenceExpression.getPrimaryConstructorParameterWithSameName(): KtParameter? {
```

Comment:
My motivation to make this method public was that there's another quickfix, that adds a constructor parameter with the same name that could be deactivated when this method returns a non-null value.